src/librustc_back/target/armv7neon_linux_androideabi.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
IMO it should probably start `thumbv7` not `armv7` because it generates Thumb code not Arm code (related https://github.com/rust-lang/rust/issues/44722)